code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -297,3 +306,29 @@
 	..()
 	if (target)
 		to_chat(user,"<span class='notice'>Tracking [target]</span>")
+
+//////////////////////////
+//pinpointer pinpointers//
+//////////////////////////
+
+/obj/item/weapon/pinpointer/pinpointerpinpointer
+	name = "pinpointer pinpointer"
+	desc = "Where did that darn pinpointer go? Hmmm... well, good thing I have this trusty pinpointer pinpointer to find it."
+	watches_nuke = FALSE
+	pinpointable = FALSE
+
+
+/obj/item/weapon/pinpointer/pinpointerpinpointer/New()
```

Comment:
You need to call `..()`

Reply:
I did call it at the end. why do I need to call it in the beginning?

Reply:
Also you didn't actually call it at the end